views/partials/_example.njk

```diff
@@ -32,13 +32,17 @@
     <li class="app-tabs__item js-tabs__item{{ " js-tabs__item--open" if (params.open) }}" role="presentation"><a href="#{{ exampleId }}-html" role="tab" aria-controls="{{ exampleId }}-html" data-track="tab-html">HTML</a></li>
     <li class="app-tabs__item js-tabs__item" role="presentation"><a href="#{{ exampleId }}-nunjucks" role="tab" aria-controls="{{ exampleId }}-nunjucks" data-track="tab-nunjucks">Nunjucks</a></li>
   </ul>
+  {% elif not (multipleTabs) and (not params.hideTab) %}
```

Comment:
I don't think we need to check for `not (multipleTabs)` here because of the above `if (multipleTabs)` logic.

Reply:
thanks, updated